lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
+ *     the response-content-disposition parameter of the signed url is set 
+ *     accordingly.
+ * @param {string=} options.responseType - If you provide this value, the 
+ *     response-content-type parameter of the signed url is set accordingly.
  * @param {function=} callback - The callback function.
  *
  * @example
  * file.getSignedUrl({
  *   action: 'read',
- *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14) // 2 weeks.
+ *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14), // 2 weeks.
+ *   responseDisposition: 'attachment; filename="filename.ext"'
```

Comment:
I left the raw header as it was more flexible but I agree with you. 
I had a look [here](http://www.iana.org/assignments/cont-disp/cont-disp.xhtm) at what values content-disposition can take and they are quite a lot. 
Explicitly handling all of them is a bit an overkill as I have the feeling that nobody uses them/they're not really supported by browsers/they're not really useful in our context. 
So I think we can go in two directions: 1) leaving it as it is (PRO: flexible, CONS: not very node-oriented, ugly) 2) name it `promptSaveAs` as you suggest and support only the `attachment` disposition (PRO: easy to understand and nice, CONS: less flexible).
Or we could allow both options and simply check which one the user specified thus providing both simplicity and flexibility. 
I'm open for suggestions!